runtime/moonbase/src/migrations.rs

```diff
@@ -0,0 +1,32 @@
+//TODO license
+
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+use pallet_author_mapping::{migrations::TwoXToBlake, Config as AuthorMappingConfig};
+use pallet_migrations::{Config as MigrationsConfig, Migration};
+use sp_std::marker::PhantomData;
+
+/// A moonbeam migration wrapping the similarly named migration in pallet-author-mapping
+pub struct AuthorMappingTwoXToBlake<T>(PhantomData<T>);
+
+impl<T: AuthorMappingConfig + MigrationsConfig> Migration for AuthorMappingTwoXToBlake<T> {
```

Comment:
My plan for managing our runtimes' migrations with pallet migrations, while still making them useable for other runtimes that don't use pallet migrations, is to have a wrapper struct for each migration struct.

Hopefully it will be less verbose in the future if we can remove the dynamic dispatch. That is still to be explored.